Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/randomText.md

> $randomText[text1;text2;etc]

Reviewer: ```suggestion
$randomText[text;...]
```

```

### Parameters

> - `text1`/`text2`/`etc` `(Type : String || Flag : Emptiable)` : The text which the bot can choose from.

Reviewer: ```suggestion
- `text` `(Type: String || Flag: Emptiable)`: The text to choose from. Separate different texts with ';'.
```
